scene/main/scene_tree.h

```diff
@@ -306,7 +306,7 @@ class SceneTree : public MainLoop {
 		for (uint32_t i = 0; i < sizeof...(p_args); i++) {
 			argptrs[i] = &args[i];
 		}
-		call_group_flagsp(p_flags, p_group, p_function, sizeof...(p_args) == 0 ? nullptr : (const Variant **)argptrs, sizeof...(p_args));
+		call_group_flagsp(BitField<GroupCallFlags>(p_flags), p_group, p_function, sizeof...(p_args) == 0 ? nullptr : (const Variant **)argptrs, sizeof...(p_args));
```

Comment:
Shouldn't `p_flags` be `BitField<GroupCallFlags>` instead of `uint32_t` in the template as well?

Reply:
As I said, this will not be presented to documentation, due to bug inside `bind_vararg_method` so I removed it. I can place it back if needed.